.github/scripts/check_labels.py

```diff
@@ -1,66 +1,22 @@
 #!/usr/bin/env python3
-"""check_labels.py"""
+"""Check a PR has required labels."""
 
-from typing import Any, List
+from typing import Any
 
-from label_utils import gh_get_labels
 from gitutils import (
     get_git_remote_name,
     get_git_repo_dir,
     GitRepo,
 )
-from trymerge import (
-    _fetch_url,
-    gh_post_pr_comment,
-    GitHubPR,
+from trymerge import GitHubPR
+from label_utils import (
+    LABEL_ERR_MSG,
+    add_label_err_comment,
+    delete_all_label_err_comments,
+    has_required_labels,
 )
 
 
-BOT_AUTHORS = ["github-actions", "pytorchmergebot", "pytorch-bot"]
-
-ERR_MSG_TITLE = "This PR needs a label"
-ERR_MSG = (
-    f"# {ERR_MSG_TITLE}\n"
-    "If your changes are user facing and intended to be a part of release notes, please use a label starting with `release notes:`.\n\n"  # noqa: E501  pylint: disable=line-too-long
-    "If not, please add the `topic: not user facing` label.\n\n"
-    "For more information, see https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work."  # noqa: E501  pylint: disable=line-too-long
-)
-
-
-def get_release_notes_labels(org: str, proj: str) -> List[str]:
-    return [label for label in gh_get_labels(org, proj) if label.lstrip().startswith("release notes:")]
-
-
-def delete_comment(comment_id: int) -> None:
-    url = f"https://api.github.com/repos/pytorch/pytorch/issues/comments/{comment_id}"
-    _fetch_url(url, method="DELETE")
-
-
-def has_required_labels(pr: GitHubPR) -> bool:
-    pr_labels = pr.get_labels()
-    # Check if PR is not user facing
-    is_not_user_facing_pr = any(label.strip() == "topic: not user facing" for label in pr_labels)
-    return (
-        is_not_user_facing_pr or
-        any(label.strip() in get_release_notes_labels(pr.org, pr.project) for label in pr_labels)
-    )
-
-
-def delete_comments(pr: GitHubPR) -> None:
```

Comment:
And since trymerge.py won't be adding comments, we don't need it to take on the deleting functionality either